openqa-mail-notification/README.md

```diff
@@ -0,0 +1,14 @@
+# OpenQA mail notification
+This script sends mails with the results of the openqa tests of OBS (Open Build Service).
```

Comment:
We should write OpenQA just like in the title to be consistent.

Reply:
Just info, the official name is openQA. Just in case you want to change all occurrences to it.